### PR TITLE
adding evidence beta flag sections to activity library and update ell section

### PR DESCRIPTION
**Pull request by emilia-friedberg**
## WHAT
Add a new "Early Access Activities" section to the activity library, which will only show for users with one of the two beta flags, and update the ELL section copy and to add a tooltip.

## WHY
We want to make it easier for teachers to find this content if they have access to it.

## HOW
We were already passing the user flag (partway) into this component, so I updated it to use `flagset` instead and made sure it came all the way down to where we needed it. Then I added the new filter and wrote the React and CSS to render it to spec.

### Screenshots
Early Access Section for evidence_beta2 users
Screen Shot 2022-09-13 at 11 26 44 AM

Early Access Section for evidence_beta1 users
Screen Shot 2022-09-13 at 2 04 45 PM

ELL section (untoggled view)
Screen Shot 2022-09-13 at 2 02 32 PM

ELL section (toggled view)
Screen Shot 2022-09-13 at 2 02 41 PM

### Notion Card Links
https://www.notion.so/quill/New-Filters-in-Activity-Library-for-Activities-in-Beta-1-Beta-2-859b2548d774433e8c31863ade0718ac
https://www.notion.so/quill/New-ELL-Tooltip-for-Activity-Library-69d163cb25b244fbaf3bc1959cd1eb8c

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? | YES
Have you deployed to Staging? | Not yet - deploying now!
Self-Review: Have you done an initial self-review of the code below on Github? | YES
Spec Review: Have you reviewed the spec and ensured this meets requirements and/or matches design mockups? | YES
Back-to-school 2022: Have you checked the [webinar schedule](https://www.notion.so/quill/Back-to-school-webinar-banners-2022-a75a89cfad9f434899ef6be3eb184733) to avoid for downtime/risky deploys? | YES

### CODE REVIEW

File: services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/custom_activity_pack/__tests__/early_access_filters.test.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import { mount } from 'enzyme';
+
+import { activities } from './data'
+
+import EarlyAccessFilters from '../early_access_filters'
+import { EVIDENCE_BETA1, EVIDENCE_BETA2, } from '../../../../../../../constants/flagOptions'
+
+function filterActivities(ignoredKey=null) { return activities }
+
+describe('AarlyAccessFilters component', () => {
```
Comment (happythenewsad) [inline services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/custom_activity_pack/__tests__/early_access_filters.test.tsx]: typo

File: services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/custom_activity_pack/early_access_filters.tsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+  )
+
+  return (
+    <Tooltip
+      isTabbable={false}
+      tooltipText={earlyAccessOption.tooltipText}
+      tooltipTriggerText={rowContent}
+    />
+  )
+}
```
Comment (anathomical) [inline services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/custom_activity_pack/early_access_filters.tsx:87]: Definitely beyond the scope of this PR, but I wonder if there's some value to abstracting `FilterRow` and even filter groups (or whatever we call what `EarlyAccessFilters` is) into something generic that gets re-used for each section.

**Review by anathomical — APPROVED** [review]
Looks good.  Approve of making a bunch of the flag strings into constants.  Some thoughts about code organization, but that's really beyond the scope of this PR.

File: services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/custom_activity_pack/filter_column.tsx [hunk 3]
```diff
@@ -98,6 +108,18 @@ const FilterColumn = ({
     />)
   }
 
+  if ([EVIDENCE_BETA1, EVIDENCE_BETA2].includes(flagset)) {
```
Comment (happythenewsad) [inline services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/custom_activity_pack/filter_column.tsx]: How about making a constant in flagOptions for this grouping? If we add EVIDENCE_BETA3 later on, this line may be missed.

**Review by happythenewsad — APPROVED** [review]
Looks great!

**Review by eadams17 — APPROVED** [review]

**Pull request merged**